mmv1/products/firebaseappcheck/firebase_app_check_service_config.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceConfig'
+base_url: projects/{{project}}/services/
+self_link: projects/{{project}}/services/{{service_id}}
+create_verb: :PATCH
+update_verb: :PATCH
```

Comment:
Could you add a handwritten update test? One that transitions from `ENFORCED` -> `OFF `-> `UNENFORCED` (and then implicitly `OFF` again during delete) would cover things pretty exhaustively.

Reply:
Added.